python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_selector_group_chat.py

```diff
@@ -192,7 +202,18 @@ async def _select_speaker(self, roles: str, participants: List[str], history: st
         num_attempts = 0
         while num_attempts < max_attempts:
             num_attempts += 1
-            response = await self._model_client.create(messages=select_speaker_messages)
+            if self._streaming:
+                message: CreateResult | str = ""
+                async for _message in self._model_client.create_stream(messages=select_speaker_messages):
+                    if isinstance(_message, LLMStreamEndEvent):
```

Comment:
`_message` actualy has the type `str` or `CreateResult`.

Reply:
You’re right, that was my mistake.
I misunderstood how `create_stream()` works and thought the log message was being leaked.

But actually, the last message is always a CreateResult, and the `if` block just `skips over LLMStreamEndEvent`, so it worked by accident.
Thanks for pointing it out!